### PR TITLE
Release CVE-2017-5264

### PR DESCRIPTION
**Pull request by todb-r7**
Updates a newly assigned CVE.

### CODE REVIEW

**Comment by cve-team** [comment]
WARNING: Commit signed, but not by a known user. Please provide your GPG key id to the CVE team.

File: 2017/5xxx/CVE-2017-5264.json [hunk 1]
```diff
@@ -1,8 +1,31 @@
 {
```
Comment (cve-team) [inline 2017/5xxx/CVE-2017-5264.json]: ERROR: Your CNA: Unknown. Owning CNA: rapid7

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
Ownership: Please see review comments for errors.

**Comment by cve-team** [comment]
Ignore the signature validation warning, Tod.

**Comment by todb-r7** [comment]
No prob! It's exciting to explore the fail conditions anyway. :)

File: 2017/5xxx/CVE-2017-5264.json (hunk 1 above)
Comment (todb-r7) [reply 2017/5xxx/CVE-2017-5264.json]: My CNA is rapid7, and todb-r7 should be associated with that.

**Pull request merged**